metadata-models/src/main/pegasus/com/linkedin/notebook/CommonCellAttributes.pdl

```diff
@@ -0,0 +1,24 @@
+namespace com.linkedin.notebook
+
+import com.linkedin.common.ChangeAuditStamps
+
+/**
+ * Common attributes for a cell in a Notebook
+ */
+record CommonCellAttributes {
+
+  /**
+   * Title of the cell
+   */
+  cellTitle: string
```

Comment:
I've seen a lot of notebooks that don't have titles on each cell. Should this be optional?

Reply:
make sense